accord-core/src/test/java/accord/impl/basic/DelayedCommandStores.java

```diff
@@ -52,6 +59,28 @@ public static CommandStores.Factory factory(PendingQueue pending)
                new DelayedCommandStores(time, agent, store, random, shardDistributor, progressLogFactory, new SimulatedDelayedExecutorService(pending, agent, random));
     }
 
+    @Override
+    protected boolean shouldBootstrap(Node node, Topology previous, Topology updated, Range range)
+    {
+        if (!super.shouldBootstrap(node, previous, updated, range))
+            return false;
+        if (!(range.start() instanceof PrefixedIntHashKey)) return true;
```

Comment:
I think this logic could be rewritten to `previous.ranges.containsAll(Ranges.of(range))` which should actually match the C* logic... can switch but wasn't sure if there would be edge cases where we don't want that as the default behavior...